R/mboostLSS.R

```diff
@@ -182,6 +182,8 @@ mboostLSS_fit <- function(formula, data = list(), families = GaussianLSS(),
             # this is the case for boosting from the beginning
             if (is.null(attr(fit, "combined_risk")) | niter == 0) {
                 combined_risk <- vapply(fit, risk, numeric(1))
+            } else {
+                combined_risk <- attr(fit, "combined_risk")()
```

Comment:
I didn't run this but wonder what this returns. Will we obtain the complete combined_risk or is it already subseted to a reasonable combined_risk vector?